docs/tutorials/pwm_control_flight_controller.rst

```diff
@@ -363,53 +390,79 @@ Reboot the flight controller and connect to it again. This time in the "Messages
+for a safety switch**.
+
+The `ArduCopter documentation <https://ardupilot.org/copter/docs/common-safety-switch-pixhawk.html>`_ provides details on how to configure the safety switch. To disable the switch,
+Set **BRD_SAFETYENABLE (also called BRD_SAFETY_DEFLT on newer firmwares) to 0**. The image below shows BRD_SAFETYENABLE set to this value in Mission Planner. If your module is
+powered on and attached to the flight controller when you change this parameter, you may hear the module arm. That is because the flight controller will begin sending 0% throttle commands
+on its outputs once this is changed.
+
+.. figure:: ../_static/tutorial_images/pwm_flight_controller/mp_brd_safety.png
+    :align: center
+
+    Disabling the Safety Switch with the BRD_SAFETYENABLE Parameter
+
+Testing the Module with Mission Planner
+*****************************************
+.. warning:: Double check that the module is secured and there is no propeller attached before performing any testing.
 
-Now we can use Mission Planner's built-in motor testing tools to make sure the flight controller can control the motor.
+Now we can use Mission Planner's built-in motor testing tools to make sure the flight controller can control the module.
 
-1. Start with the motor powered off, but connected to the flight controller.
+1. Start with the module powered off, but connected to the flight controller.
 2. Select "Setup" from the top toolbar, and then expand "Optional Hardware" on the sidebar
-3. Select "Motor Test" from the "Optional Hardware" options.
-4. Power on the motor and wait for it to complete its startup song.
+3. Select "Motor Test" from the "Optional Hardware" options, see the image below as an example of what the Motor Test screen should look like.
+
+  .. figure:: ../_static/tutorial_images/pwm_flight_controller/mp_motor_test.png
+    :align: center
+
+    Motor Test Screen in Mission Planner
+
+4. Power on the module and wait for it to complete its startup song.
+
+  * If you have disabled the safety switch and are using Standard PWM, the module should play its arming song after startup when attached to the flight controller.
+
 5. Arm your safety switch if you have one.
    
-  * If using PWM, the motor should play its two-note arming song as the flight controller starts sending 0% throttle commands once the safety switch is armed
-  * On DSHOT, the motor will not arm yet.
+  * If using Standard PWM, the module should play its two-note arming song as the flight controller starts sending 0% throttle commands once the safety switch is armed
+  * On DSHOT, the module will not arm yet.
   
-6. Set the "Throttle %" to 3% and the "Duration" to 5s, and click "Test All Motors" 
-7. The motor should spin slowly for 5 seconds, and then stop.
+6. Set the "Throttle %" to 5% and the "Duration" to 5s, and click "Test All Motors" 
```

Comment:
see above about periods. double check through your lists/bullets that there's a consistent style